docs/src/main/paradox/sqs.md

```diff
@@ -194,12 +194,47 @@ Java (change visibility timeout)
 Same as the normal `SqsSink`:
 
 Scala
-: @@snip ($alpakka$/sqs/src/main/scala/akka/stream/alpakka/sqs/SqsAckSinkSettings.scala) { #SqsAckSinkSettings }
+: @@snip ($alpakka$/sqs/src/main/scala/akka/stream/alpakka/sqs/SqsAckSettings.scala) { #SqsAckSinkSettings }
 
 Options:
 
  - `maxInFlight` - maximum number of messages being processed by `AmazonSQSAsync` at the same time. Default: 10
 
+### Message processing with acknowledgement with underlying batching
+
+`SqsAckSink.grouped` a sink that can acknowledge (delete), ignore, or postpone messages, but it batches items and sends them as one request per action.
+
+Scala (ack)
+: @@snip ($alpakka$/sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSpec.scala) { #batch-ack }
+
+Scala (ignore)
+: @@snip ($alpakka$/sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSpec.scala) { #batch-ignore }
+
+Scala (change visibility timeout)
+: @@snip ($alpakka$/sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSpec.scala) { #batch-requeue }
+
+Java (ack)
+: @@snip ($alpakka$/sqs/src/test/java/akka/stream/alpakka/sqs/javadsl/SqsAckSinkTest.java) { #batch-ack }
+
+Java (ignore)
+: @@snip ($alpakka$/sqs/src/test/java/akka/stream/alpakka/sqs/javadsl/SqsAckSinkTest.java) { #batch-ignore }
+
+Java (change visibility timeout)
+: @@snip ($alpakka$/sqs/src/test/java/akka/stream/alpakka/sqs/javadsl/SqsAckSinkTest.java) { #batch-requeue }
+
```

Comment:
How do these 6 tabs render on the page? It might be better to group them Scala/Java per use case.

Reply:
Any example on how to do it?

Reply:
I mean the normal 
```
Scala
: @@snip (....

Java
: @@snip (...
```
and some text between each pair.